tensorflow/core/distributed_runtime/rpc/grpc_call.h

```diff
@@ -247,10 +247,10 @@ class Call : public UntypedCall<Service> {
 
   // Used as void* completion markers from grpc to indicate different
   // events of interest for a Call.
-  using typename UntypedCall<Service>::Tag;
-  Tag request_received_tag_{this, Tag::kRequestReceived};
-  Tag response_sent_tag_{this, Tag::kResponseSent};
-  Tag cancelled_tag_{this, Tag::kCancelled};
+  typedef typename UntypedCall<Service>::Tag _TTag;
```

Comment:
Does the following work?

```c++
typedef typename UntypedCall<Service>::Tag Tag;
```

...and then can you revert the next three lines? (I don't have a copy of GCC 4.8.2 handy to test this.)